static/scripts/rewards/gift-cards/gift-card.ts

```diff
@@ -0,0 +1,116 @@
+import { BigNumberish } from "ethers";
+import { GiftCard } from "../../../../shared/types";
+import { getFixedPriceToValueMap, getGiftCardValue, getRangePriceToValueMap, isRangePriceGiftCardClaimable } from "../../../../shared/pricing";
+import { formatEther } from "ethers/lib/utils";
+
+const html = String.raw;
```

Comment:
String.raw is new for me. Is that essentially initializing a string buffer I.e. "" ?

Reply:
It's used to get the raw string form of template literals. The point of having this is to make prettier format the html in template literals. https://stackoverflow.com/a/73068825/2276036
https://github.com/ubiquity/pay.ubq.fi/blob/8fa0b09c8d372688f5a2984632e0ec8a0b66d1eb/static/scripts/rewards/gift-cards/gift-card.ts#L9-L10